src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -465,6 +465,12 @@
     @Parameter( property = "license.sortArtifactByName", defaultValue = "false" )
     boolean sortArtifactByName;
 
+    /**
+     * A flag to create the third-party file as a csv file
```

Comment:
This comment is not informative enough. It should say clearly what happens if the value is `true` and what happens if the value is `false`. Good style: "If `true` does x, otherwise does y".

BTW, where will the CSV file going to get stored? Should there not be an option for that? Can such a file path option not replace this `createCSV` flag? I mean `csvPath` being null clearly implies `createCSV = false`.